modules/transfers/src/domain/transfer_entity.ts

```diff
@@ -58,11 +66,24 @@ export class TransferEntity extends BaseEntity<TransferState> {
     return entity
   }
 
-  setupInitialState (initialState: TransferState): void {
-    this._state = { ...initialState }
+  prepareTransfer (incominmgTransfer: PrepareTransferData): void {
```

Comment:
Should we not pass the Cmd as the input here until we have some kind of common data model?

Also, note the spelling of `incominmgTransfer`